zingo-account/src/lib.rs

```diff
@@ -0,0 +1,28 @@
+//! A micro crate to provide a wrapper around the zcash_client_backend account trait.
+//! By constraining the wrapper to be independent of internal zingolib functionality
+//! we expose a shareable component, and refine the definition of zingolib.
+
+/// The a view-only abstraction that provides a management interface for
+/// TODO:  Explain more about our Account abstraction.
+pub struct ZingoAccount(
+    pub zip32::AccountId,
+    pub zcash_keys::keys::UnifiedFullViewingKey,
```

Comment:
I dont think we should make fields pub unless there is a really good reason? is there one?
the `id` and `ufvk` methods are private though which is surprising. Have you looked at how this Account trait is used in LRZ?